src/infiniteGrid.js

```diff
@@ -454,7 +254,7 @@ eg.module("infiniteGrid", ["jQuery", eg, window, document, "Outlayer"], function
 		 * @return {Boolean}
 		 */
 		isRecycling: function() {
-			return this.core.options.count > 0 && this._isRecycling;
+			return (this.options.count > 0) && this._isRecycling;
```

Comment:
This is same to `isRecycling()`.
Overall, both of 1_isRecycling1 and 1isRecycling()1 called. I think only `isRecyling()` shuld used.